packages/dd-trace/src/log.js

```diff
@@ -7,9 +7,29 @@ const _default = {
   error: err => console.error(err) /* eslint-disable-line no-console */
 }
 
+const _logLevels = {
+  'debug': 0,
+  'error': 1
+}
+
+const _defaultLogLevel = 'debug'
+
 let _logger
 let _enabled
 let _deprecate
+let _logLevel
+
+const _isLogLevelEnabled = (level) => {
+  return !_logLevel || _logLevels[level] >= _logLevel
+}
+
+const _setLogLevel = (logLevel) => {
```

Comment:
This function is not setting a value, so the name `set` is misleading.

Reply:
updated to `_checkLogLevel`